config.js

```diff
@@ -12,7 +12,7 @@ const setHeaders = res => {
 }
 
 const mongo_connect = () => {
-    const MONGO_URL = 'mongodb://admin:admin123@ds231537.mlab.com:31537/bridgeit';
+    const MONGO_URL = 'mongodb+srv://Maciek:Antek@gabriel-jtivb.mongodb.net/test?retryWrites=true';

```

Comment:
I think we have to move these configuration to .env. I mean hide for users. We have to talk about it